assets/js/tooltips/tooltips.js

```diff
@@ -0,0 +1,321 @@
+// Dependencies
+// ------------
+
+import $ from 'jquery'
+import tooltipBodyTemplate from '../templates/tooltip-body.handlebars'
+import tooltipLayoutTemplate from '../templates/tooltip-layout.handlebars'
+
+// Constants
+// ---------
+const footerSelector = 'footer' // `Enable/Disable tooltips` button will be displayed in the footer
+const tooltipActivators = '.content a code, .detail-header .specs a' // Elements that can activate the tooltip
+const tooltipSelector = '#tooltip' // Tooltip root
+const tooltipIframeSelector = '#tooltip .tooltip-iframe' // Iframe inisde the tooltip, will be used to load external pages
+const contentInner = 'body .content-inner' // Element containing the documentation text
+const spacingBase = 10 // Used as the min. distance from window edges and links
+const minBottomSpacing = spacingBase * 5 // Min. space needed between the bottom of the bottom of the page and the bottom edge of the tooltip
+const minWindowSize = { // Tooltips won't be displayed if width/height of the viewport is smaller than this
+  height: 450,
+  width: 768
+}
+// Tooltip will appear only if the mouse cursor stays on the link for at least 150ms.
+// This way tooltips will not appear if we are scrooling the page or just moving the cursor around.
+const hoverDelayTime = 150
```

Comment:
Because there is a loading time, I think we should be fine with 100 here. :D (bike shedding)